pkg/cloudprovider/provider/anexia/types/types.go

```diff
@@ -42,11 +40,6 @@ const (
 	MachinePoweredOn = "poweredOn"
 )
 
-var StatusUpdateFailed = cloudprovidererrors.TerminalError{
-	Reason:  common.UpdateMachineError,
-	Message: "Failed to update the machine status",
-}
```

Comment:
I _suspect_ this is something you did because of linting errors, can we please just quickly document (in this thread) why it's being removed?